python/src/spark_rapids_ml/connect_plugin.py

```diff
@@ -115,20 +113,44 @@ def main(infile: IO, outfile: IO) -> None:
         # Create DataFrame
         df = DataFrame(jdf, spark)
 
-        print(f"Running {estimator_name} with parameters: {params}")
+        print(f"Running {operator_name} with parameters: {params}")
         params = json.loads(params)
 
-        if estimator_name == "LogisticRegression":
-            from .classification import LogisticRegression
+        if operator_name == "LogisticRegression":
+            from .classification import LogisticRegression, LogisticRegressionModel
 
             lr = LogisticRegression(**params)
             model: LogisticRegressionModel = lr.fit(df)
             model_cpu = model.cpu()
             assert model_cpu._java_obj is not None
             model_targe_id = model_cpu._java_obj._get_object_id().encode("utf-8")
             write_with_length(model_targe_id, outfile)
+            # Model attributes
+            attributes = [
+                model.coef_,
+                model.intercept_,
+                model.classes_,
+                model.n_cols,
+                model.dtype,
+                model.num_iters,
+                model.objective,
+            ]
+            write_with_length(json.dumps(attributes).encode("utf-8"), outfile)
+
+        elif operator_name == "LogisticRegressionModel":
+            attributes = utf8_deserializer.loads(infile)
+            attributes = json.loads(attributes)  # type: ignore[arg-type]
+            from .classification import LogisticRegression, LogisticRegressionModel
+
+            # Workaround for setting params for corresponding model
```

Comment:
We could enable this more easily in spark rapids ml code if needed but this seems to work ok.

Reply:
Done, the issue has been fixed by _set_param API in both estimator and model.